edgeless_node/Cargo.toml

```diff
@@ -53,6 +53,7 @@ prost = "0.12"
 base64 = "0.21.5"
 rs-docker = "0.0.58"
 chrono = "0.4.38"
+ollama-rs = { version = "0.2.0", features = ["chat-history"] }
```

Comment:
This is not specific to this resource, but given we are getting many more specialized resources we should modularize this so the default build does not need to depend on stuff like that.